public/views/content/entryForm.html

```diff
@@ -4,7 +4,7 @@
     <div class="col-sm-6">
       <div class="form-group">
         <label for="date">Date:</label>
-        <input type="date" id="date" class="form-control" ng-model="newEntry.date" value="{{ date | date: 'yyyy-MM-dd' }}" />
+        <input data-provide="datepicker" type="text" class="form-control datepicker" ng-model="newEntry.date" value="{{ date | date: 'mm/dd/yyyy' }}">
```

Comment:
Found the solution. You just need to change the Date type to Number type in formSchema